karols-frontend/src/components/CardModel.js

```diff
@@ -100,7 +105,7 @@ const CardModel = ({
                   transform: "translateY(calc(0px + 1em))"
                 }}
               >
-                {description}
+                {description} <Counter />
```

Comment:
Counter need to be displayed only if showCounter is true, and need the appropriate props